File: Moving Zeros To The End.py
def move_zeros(array):
    a = []
    count = 0
    
    for i in array:
        if i == 0:
            count += 1
        else:
            a.append(i)
    
    for i in range(count):
        a.append(0)
    
    return a

File: Moving Zeros To The End.js
function moveZeros(arr) {
  var count = 0
  var a = []
  
  for(let i = 0; i < arr.length; i++){
    if(arr[i] === 0){
      count++
    } else {
      a.push(arr[i])
    }
  }
  
  for(let j = 0; j < count; j++){
    a.push(0)
  }
  
  return a
}
